pkg/cloud/gomock_reflect_1599216259/prog.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
```

Comment:
Pretty sure we don't need this file here. It's generated for Mocks right?

If I'm correct, please remove it and add a .gitignore entry. Something like `pkg/cloud/gomock_reflect_*`